error.go

```diff
@@ -86,6 +86,27 @@ func IsNotFound(err error) bool {
 	return microerror.Cause(err) == notFoundError
 }
 
+var (
+	noHostErrorString = "no such host"
+)
+
+var noHostError = &microerror.Error{
+	Kind: "noHostError",
+}
+
+// IsNoHostError asserts noHostError.
+func IsNoHostError(err error) bool {
+	if err == nil {
+		return false
+	}
+
+	if err == noHostError {
+		return true
+	}
+
+	return strings.Contains(err.Error(), noHostErrorString)
+}
```

Comment:
This should go to `error_internal.go` as `isNoHostError` w/o corresponding `noHosError` variable. See https://github.com/giantswarm/fmt/blob/master/go/errors.md#handling-third-party-library-errors